client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx

```diff
@@ -25,7 +25,7 @@ import {
   getLocationInputColumn,
   InventoryAdjustmentReasonRowFragment,
   InventoryAdjustmentReasonSearchInput,
-  ItemVariantInputCell,
+  ItemVariantInputCellOld,
```

Comment:
Keeping old by renaming... should be removed once we've replaced them all

Reply:
Can we add this to one of the follow up issues, so we don't forget?

Reply:
https://github.com/msupply-foundation/open-msupply/issues/7630